azure-pipelines.yml

```diff
@@ -200,6 +206,7 @@ jobs:
 
 
   - script: |
+      set /p ARCH=$(Platform)
```

Comment:
```suggestion
      set ARCH=$(Platform)
```

`/p` means get value from user input, thus this env is missing.

Reply:
BTW, L210 works because it uses `<` to read from a file.